internal/env/env.go

```diff
@@ -1,10 +1,61 @@
 package env
 
 import (
+	"errors"
+	"fmt"
 	"os"
+	"strconv"
 	"strings"
 )
 
+const (
+	// ------------------------
+	//          auth
+	// ------------------------
+
+	// EnvVarAuthToken is the env var for the auth token sent by the launcher to
+	// the main instance in exchange for a single-use grant token.
+	// nolint:gosec // G101: False positive
+	EnvVarAuthToken = "N8N_RUNNERS_AUTH_TOKEN"
+
+	// EnvVarGrantToken is the env var for the single-use grant token returned by
+	// the main instance to the launcher in exchange for the auth token.
+	// nolint:gosec // G101: False positive
+	EnvVarGrantToken = "N8N_RUNNERS_GRANT_TOKEN"
+
+	// ------------------------
+	//        n8n main
+	// ------------------------
+
+	// EnvVarMainServerURI is the env var for the URI of the n8n main instance's
+	// main server.
+	EnvVarMainServerURI = "N8N_MAIN_URI"
+
+	// EnvVarMainRunnerServerURI is the env var for the URI of the n8n main
+	// instance's runner server. Do not confuse with `N8N_RUNNER_URI`.
+	EnvVarMainRunnerServerURI = "N8N_RUNNERS_N8N_URI"
+
+	// ------------------------
+	//         runner
+	// ------------------------
+
+	// EnvVarRunnerServerURI is the env var for the URI of the runner's server. Do not
+	// confuse with `N8N_RUNNERS_N8N_URI`.
+	EnvVarRunnerServerURI = "N8N_RUNNER_URI"
+
+	// EnvVarRunnerServerEnabled is the env var for whether the runner's health
+	// check server should be started.
+	EnvVarRunnerServerEnabled = "N8N_RUNNERS_SERVER_ENABLED"
```

Comment:
Maybe we could simplify and always start the runner's healthcheck server. We could get rid of on of the env vars. Can be a followup

Reply:
Good idea!